File: app/src/main/java/com/uralsiberianworks/neuralpushkin/BaseActivity.java
package com.uralsiberianworks.neuralpushkin;

import android.widget.TextView;


import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;


public class BaseActivity extends AppCompatActivity {

    public final void setupToolbar(int toolbarId, String titlePage){
        Toolbar toolbar = (Toolbar) findViewById(toolbarId);
        setSupportActionBar(toolbar);

        TextView title = (TextView) toolbar.findViewById(R.id.tv_title);
        title.setText(titlePage);

        getSupportActionBar().setTitle("");
    }

}

File: app/src/main/java/com/uralsiberianworks/neuralpushkin/BaseActivity.kt
package com.uralsiberianworks.neuralpushkin

import android.view.View
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar

open class BaseActivity : AppCompatActivity() {
    fun setupToolbar(toolbarId: Int, titlePage: String?) {
        val toolbar = findViewById<View>(toolbarId) as Toolbar
        setSupportActionBar(toolbar)
        val title = toolbar.findViewById<View>(R.id.tv_title) as TextView
        title.text = titlePage
        supportActionBar!!.title = ""
    }
}